lib/elix/lists.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Elix.Lists do
+  # TODO: prevent duplicate lists
+  # TODO: sanitize list names
+  # TODO: error handling
+
+  @all_lists_key "lists"
+
+  def all do
+    {:ok, lists} = Redix.command(:redix, ["LRANGE", @all_lists_key, 0, -1])
+    lists
+  end
+
+  def create(list_name) do
+    {:ok, _length} = Redix.command(:redix, ["RPUSH", @all_lists_key, list_name])
+    all
+  end
+
+  def delete(list_name) do
+    {:ok, _length} = Redix.command(:redix, ["LREM", @all_lists_key, 0, list_name])
+    all
+  end
+
+end
```

Comment:
I didn't do this with the GifMeresponder, it felt like a good idea to put the responsibility of actually managing lists into a separate module from the responder, so the responder can just worry about input parsing and delegation.